.licenserc.yaml

```diff
@@ -27,4 +27,9 @@ header:
     - 'src/loki_alert_rules/.gitkeep'
     - 'src/prometheus_alert_rules/content_cache_missing.rule'
     - 'content-cache_rock/nginx_cfg.tmpl'
+    - 'files/logrotate.conf'
+    - 'files/rsyslog.conf'
+    - 'files/usr.bin.pollen'
```

Comment:
I think if that doesn't block it from running we definitely should have the license headers on these

Reply:
If not, you can add custom license config for each type of file. usr.bin.pollen does actually have a license header.